### PR TITLE
WEA-2 Add Firebase Crashlytics

### PR DESCRIPTION
**Pull request by ayush-shta**
Firebase Crashlytics setup

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -31,6 +32,10 @@ dependencies {
   implementation 'com.android.support.constraint:constraint-layout:1.1.2'
 
   implementation 'com.jakewharton:butterknife:8.8.1'
+  implementation 'com.google.firebase:firebase-crash:11.8.0'
```
Comment (subash-poudel) [inline app/build.gradle]: wrong place between similar library

Comment (ayush-shta) [reply app/build.gradle]: :ok:

File: app/google-services.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (subash-poudel) [inline app/google-services.json]: Remove from public repo see this link:https://stackoverflow.com/questions/37358340/should-i-add-the-google-services-json-from-firebase-to-my-repository

File: build.gradle [hunk 3]
```diff
@@ -5,19 +5,28 @@ buildscript {
   repositories {
     google()
     jcenter()
+    maven {
+      url 'https://maven.fabric.io/public'
+    }
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.1.3'
+    classpath 'com.android.tools.build:gradle:3.0.0'
```
Comment (subash-poudel) [inline build.gradle]: why is gradle downgraded?

File: build.gradle [hunk 4]
```diff
@@ -5,19 +5,28 @@ buildscript {
...
 
     //old version for compatibility with stetho
     classpath "io.realm:realm-gradle-plugin:3.0.0"
+    classpath 'com.google.gms:google-services:3.1.1'
```
Comment (subash-poudel) [inline build.gradle:18]: write a comment for which library you need this

File: app/build.gradle [hunk 5]
```diff
@@ -31,6 +32,10 @@ dependencies {
   implementation 'com.android.support.constraint:constraint-layout:1.1.2'
 
   implementation 'com.jakewharton:butterknife:8.8.1'
+  implementation 'com.google.firebase:firebase-core:11.8.0'
+  implementation 'com.google.firebase:firebase-crash:11.8.0'
+  implementation 'com.crashlytics.sdk.android:crashlytics:2.9.3'
+
   annotationProcessor 'com.jakewharton:butterknife-compiler:8.8.1'
```
Comment (subash-poudel) [inline app/build.gradle]: this should be below   implementation 'com.jakewharton:butterknife:8.8.1'

**Comment by subash-poudel** [comment]
👍

**Pull request merged**